.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1296,11 +1328,14 @@ public class ChatMessageContentPart : IJsonModel<ChatMessageContentPart>, IPersi
         public string ImageBytesMediaType { get; }
         public ChatImageDetailLevel? ImageDetailLevel { get; }
         public Uri ImageUri { get; }
+        public BinaryData InputAudioBytes { get; }
+        public ChatInputAudioFormat? InputAudioFormat { get; }
         public ChatMessageContentPartKind Kind { get; }
         public string Refusal { get; }
         public string Text { get; }
         public static ChatMessageContentPart CreateImagePart(BinaryData imageBytes, string imageBytesMediaType, ChatImageDetailLevel? imageDetailLevel = null);
         public static ChatMessageContentPart CreateImagePart(Uri imageUri, ChatImageDetailLevel? imageDetailLevel = null);
+        public static ChatMessageContentPart CreateInputAudioPart(BinaryData audioBytes, ChatInputAudioFormat audioFormat);
```

Comment:
nit: We should rename the parameters too: `inputAudioBytes`, and `inputAudioFormat`.